contributors/design-proposals/device-plugin.md

```diff
@@ -1,99 +1,77 @@
-# Device Manager Proposal
-
-  1. [Abstract](#abstract)
-  2. [Motivation](#motivation)
-  3. [Use Cases](#use-cases)
-  4. [Objectives](#objectives)
-  5. [Non Objectives](#non-objectives)
-  6. [Stories](#stories)
-      * [Vendor story](#vendor-story)
-      * [User story](#user-story)
-  8. [Device Plugin](#device-plugin)
-      * [Protocol Overview](#protocol-overview)
-      * [Protobuf specification](#protobuf-specification)
-      * [Installation](#installation)
-      * [API Changes](#api-changes)
-      * [Versioning](#versioning)
+Device Manager Proposal
+===============
+
+<!-- BEGIN MUNGE: GENERATED_TOC -->
+
+- [Motivation](#motivation)
+- [Use Cases](#use-cases)
+- [Objectives](#objectives)
+- [Non Objectives](#non-objectives)
+- [Proposed Implementation 1](#proposed-implementation-1)
+  - [Vendor story](#vendor-story)
+  - [End User story](#end-user-story)
+  - [Device Plugin](#device-plugin)
+    - [Introduction](#introduction)
+    - [Registration](#registration)
+    - [Unix Socket](#unix-socket)
+    - [Protocol Overview](#protocol-overview)
+    - [Protobuf specification](#protobuf-specification)
+- [Proposed Implementation 2](#proposed-implementation-2)
+  - [Device Plugin Lifecycle](#device-plugin-lifecycle)
+  - [Protobuf API](#protobuf-api)
+  - [Failure recovery](#failure-recovery)
+  - [Roadmap](#roadmap)
+  - [Open Questions](#open-questions-1)
+- [Installation](#installation)
+- [Versioning](#versioning)
+  - [References](#references)
+
+<!-- END MUNGE: GENERATED_TOC -->
```

Comment:
This is auto generated by mungedocs tool. If you want, we can remove them in the final versions of the doc, but I saw some docs in the repo just keep them there.